egov-notification-sms/README.md

```diff
@@ -1,78 +1,22 @@
-Notification SMS services consumes SMS from the kafka notification topic and 
-process them to send it to an external 3rd party service.
+# egov-notification-sms service
 
-The implementation of the consumer is present in the directory `src/main/java/org/egov/web/notification/sms/service/impl`.
+Notification SMS service consumes SMS from the kafka notification topic and process them to send it to an third party service.
 
-There are current below providers available
-- Generic
-- Console
-- MSDG
+### DB UML Diagram
 
-The implementation to be used can be configured by setting `sms.provider.class`. 
+- NA
 
-### Console
+### Service Dependencies
+- NA
 
-The `Console` implementation just prints the message mobile number and message to the console
+### Swagger API Contract
 
-### Generic implementation
+- NA
 
-This is the default implementation, which can work with most of the SMS Provider. The generic implementation supports below
-- GET or POST based API
-- Supports query params, form data, JSON Body
+## Service Details
 
-To configure the url of the SMS provider use `sms.provider.url` property
-To configure the http method used configure the `sms.provider.requestType` property to either `GET` or `POST`.
-
-To configure form data or json api set `sms.provider.contentType=application/x-www-form-urlencoded` or `sms.provider.contentType=application/json` respectively
-
-To configure which data needs to be sent to the API below property can be configured:
-
-- `sms.config.map`={'uname':'$username', 'pwd': '$password', 'sid':'$senderid', 'mobileno':'$mobileno', 'content':'$message', 'smsservicetype':'unicodemsg', 'myParam': '$extraParam' , 'messageType': '$mtype'}
-- `sms.category.map`={'mtype': {'*': 'abc', 'OTP': 'def'}}
-- `sms.extra.config.map`={'extraParam': 'abc'}
-
-`sms.extra.config.map` is not used currently and is only kept for custom implementation which requires data that doesn't need to be directly passed to the REST API call
-
-`sms.config.map` is a map of parameters and their values
-
-Special variables that are mapped
-
-- `$username` maps to `sms.provider.username`
-- `$password` maps to `sms.provider.password`
-- `$senderid` maps to `sms.senderid`
-- `$mobileno` maps to `mobileNumber` from kafka fetched message
-- `$message` maps to the `message` from the kafka fetched message
-- `$<name>` any variable that is not from above list, is first checked in `sms.category.map` and then in `application.properties` and then in environment variable with full upper case and `_` replacing `-`, space or `.`
-
-So if you use `sms.config.map={'u':'$username', 'p':'password'}`. Then the API call will be passed `<url>?u=<$username>&p=password`
-
-
-#### Message Success or Failure
-
-Message success delivery can be controlled using below properties
```

Comment:
This is an important section and cannot be deleted. Either all these details need to be move to confluence